app/src/main/res/values/strings.xml

```diff
@@ -114,6 +114,8 @@
     <string name="all_review_rating">Rating: %.1f</string>
     <string name="all_review_text">Text: %s</string>
     <string name="undefined_username">Unknown user</string>
+    <string name="sort_by_rating">Sort By Rating (Best Rated First)</string>
+    <string name="sort_by_date">Sort By Date (Most Recent First)</string>
```

Comment:
this is most likely just nitpicking but I am uncomfortable with capping the begining of each word when it's not necessary

Reply:
This is a stylistic choice, and one that respects conventions to my knowledge, so this should probably be handled at a later time